pkg/api/groups.go

```diff
@@ -558,9 +558,10 @@ func (api *API) GetGroupStatusCountTimeline(groupID string, duration string) (ma
 	}
 	// Get the versions and their number of instances per status within each of the given time intervals.
 	query := fmt.Sprintf(`
-	WITH time_series AS (SELECT * FROM generate_series(now() - interval '%[1]s', now(), INTERVAL '%[2]s') AS ts)
+	WITH time_series AS (SELECT * FROM generate_series(now() - interval '%[1]s', now(), INTERVAL '%[2]s') AS ts),
+	min_time AS (select min(ts) from time_series)
```

Comment:
I'd make some keywords uppercase and I'd name the column explicitly. I'd pick a name like `min_ts` to avoid confusion with the `min` function:

```suggestion
	min_time AS (SELECT min(ts) AS min_ts FROM time_series)
```